layers.go

```diff
@@ -772,7 +772,18 @@ func (r *layerStore) Mounted(id string) (int, error) {
 }
 
 func (r *layerStore) Mount(id string, options drivers.MountOpts) (string, error) {
-	if !r.IsReadWrite() {
+
+	// check whether options include ro option
+	hasReadWriteOpt := func(opts []string) bool {
+		for _, item := range opts {
+			if item == "ro" {
+				return false
+			}
+		}
+		return true
+	}
+
+	if !r.IsReadWrite() && hasReadWriteOpt(options.Options) {
```

Comment:
Can you add a comment here that explains the same as in the commit message? Otherwise, I will likely forget by next week :)